OMGVA PoS/ClientApp/src/pages/SelectBusinessPage.tsx

```diff
@@ -0,0 +1,65 @@
+﻿import React, { useState, useEffect } from 'react';
+import { useNavigate } from 'react-router-dom'
+import { getAllBusinesses } from '../services/businessService';
+import ClickableListItem from '../components/List/ClickableListItem';
+import '../index.css';
+import '../components/List/ClickableListItem.css';
+import { getTokenRole } from '../utils/tokenUtils';
+
+interface SelectBusinessPageProps {
+    token : string | null
+}
+
+const SelectBusinessPage: React.FC<SelectBusinessPageProps> = ({token: authToken}) => {
+    const [listItems, setListItems] = useState<Array<JSX.Element>>();
+    const [error, setError] = useState<string | null>(null);
+    const navigate = useNavigate();
+
+    const getBusinesses = async () => {
+        setError(null);
+
+        try {
+            const { result, error } = await getAllBusinesses(authToken);
+
+            if (!result) {
+                setError('Problem acquiring businesses: ' + error);
```

Comment:
Maybe let's not show client the whole error? I think that 'Problem acquiring businesses' is as much informative as it needs to be

Reply:
The error is usually http response code. I find it kind of useful